docs/hw/display/nv3/pramdac.rst

```diff
@@ -18,12 +18,16 @@ Introduction
 MMIO registers
 ==============
 
-.. space:: 8 pramdac 0x1000 RAMDAC, video overlay, cursor, and PLL control
+.. space:: 32 pramdac 0x1000 RAMDAC, video overlay, cursor, and PLL control
 
-   .. todo:: write me
+   0x500 NVPLL nv4-pramdac-nvpll
+   0x504 MPLL nv4-pramdac-mpll
+   0x508 VPLL nv4-pramdac-vpll
 
 .. space:: 8 prmdio 0x1000 VGA DAC registers
 
    .. todo:: write me
 
-.. todo:: write me
+The bit layout for all NV4 PLLs is that bits 18-16 are P, bits 15-8 are N, and bits 7-0 are M.
```

Comment:
Please leave a todo item here: it's missing at least information about valid N/M/P ranges.